apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/helpers.php

```diff
@@ -192,3 +192,36 @@ function populate_wp_template_data() {
 }
 register_activation_hook( __FILE__, __NAMESPACE__ . '\populate_wp_template_data' );
 add_action( 'switch_theme', __NAMESPACE__ . '\populate_wp_template_data' );
+
+/**
+ * Shows a confirm prompt when the plugin is about to be deactivated on a unlaunched site.
+ *
+ * This will filter the FSE actions on the plugin manager list to add the confirm
+ * prompt on the click event of the action=deactivate.
+ *
+ * The option launch-status exists only on wpcom sites.
+ */
+add_filter(
+	'plugin_action_links_full-site-editing/full-site-editing-plugin.php',
```

Comment:
It would be better to get the plugin name dynamically.

```php
'plugin_action_links_' . plugin_basename( $this->get_plugin_file() )
```

Reply:
`$this->get_plugin_file()` throws this error:
<img width="621" alt="Screen Shot 2565-11-15 at 17 38 51" src="https://user-images.githubusercontent.com/1881481/201898850-1cec0e04-57fd-4809-938f-cc7851cfb5fc.png">

I haven't found documentation on `get_plugin_file()`. Any ideas?

Reply:
Sorry, I forgot to replace it with the real one. It's `__FILE__`.

Reply:
Thank you! I tested it, and it worked as expected. 
I noticed that when uploading the plugin the directory name ends with `-1`, like this `full-site-editing-1/full-site-editing-plugin.php`